studio/components/interfaces/Auth/RateLimits/RateLimits.tsx

```diff
@@ -0,0 +1,336 @@
+                    Rate limit for sending SMS messages
+                  </FormSectionLabel>
+                }
+              >
+                <FormSectionContent loading={false}>
+                  <FormField_Shadcn_
+                    control={form.control}
+                    name="RATE_LIMIT_SMS_SENT"
+                    render={({ field }) => (
+                      <FormItem_Shadcn_>
+                        <FormControl_Shadcn_>
+                          <Input_Shadcn_
+                            disabled={!canUpdateSMSRateLimit}
+                            type="number"
+                            {...field}
+                          />
+                        </FormControl_Shadcn_>
+                        <FormMessage_Shadcn_ />
+                        {!canUpdateSMSRateLimit && (
+                          <Alert_Shadcn_>
+                            <IconAlertCircle strokeWidth={1.5} />
+                            <AlertTitle_Shadcn_>
+                              Enable phone-based logins to update this configuration
+                            </AlertTitle_Shadcn_>
+                            <AlertDescription_Shadcn_>
+                              <p className="!leading-tight">
+                                Head over to the providers page to enable phone provider and phone
+                                confirmations before updating your rate limit
+                              </p>
+                              <Link passHref href={`/project/${projectRef}/auth/providers`}>
+                                <Button asChild type="default" className="mt-2">
+                                  <a>View providers configuration</a>
+                                </Button>
+                              </Link>
+                            </AlertDescription_Shadcn_>
+                          </Alert_Shadcn_>
+                        )}
+                      </FormItem_Shadcn_>
+                    )}
+                  />
+                </FormSectionContent>
+              </FormSection>
+
+              <FormSection
+                id="token-refresh"
+                header={
+                  <FormSectionLabel
+                    description={
+                      <p className="text-foreground-light text-sm">
+                        How many sessions can be refreshed in a 5 minute interval
+                      </p>
+                    }
+                  >
+                    Rate limit for token refreshes
+                  </FormSectionLabel>
+                }
+              >
+                <FormSectionContent loading={false}>
+                  <FormField_Shadcn_
+                    control={form.control}
+                    name="RATE_LIMIT_TOKEN_REFRESH"
+                    render={({ field }) => (
+                      <FormItem_Shadcn_>
+                        <FormControl_Shadcn_>
+                          <Input_Shadcn_ disabled type="number" {...field} />
```

Comment:
Why is this disabled?

Reply:
The 2 fields are not exposed through API yet, hence why they're values are hard coded for now too
Needs some BE work before users can self-serve